sklearn/preprocessing/tests/test_encoders.py

```diff
@@ -265,6 +265,16 @@ def test_one_hot_encoder_inverse(sparse_, drop):
         enc.inverse_transform(X_tr)
 
 
+def test_one_hot_encoder_inverse_if_binary():
+    X = [['Male', 1],
```

Comment:
We can make the input a numpy array:

```py
    X = np.array([['Male', 1],
                  ['Female', 3],
                  ['Female', 2]])
```

and then

```py
assert_array_equal(ohe.inverse_transform(X_tr), X)
```

Reply:
done